operations/messaging-and-math/content-marketing/video-content.md

```diff
@@ -0,0 +1,86 @@
+---
+description: '50% DRAFT (revised by Ashley Dotterweich on Oct 20, 2020)'
+---
+
+# Video Content Best Practices
+
+This document provides guidelines for creating video content for Mattermost. It covers:
+
+* What is video content used for?
+* How can I create great video content?
+* What is the process for developing video content?
+* What is the process for publishing video content?
+* How is video content promoted?
+
+## The Pros and Cons of Video Content
+
+**What is video content used for?**
+
+Video content provides a dynamic experience for viewers and can add personality and context to a topic where written content cannot. We should use video content when video provides the best audience experience for a piece of content. Some examples include discussions between two or more people and tutorials with complex or highly visual workflows.
+
+Videos can also be helpful for repurposing live events and sessions for longer term use. This includes presentations at live events and internal product demos.
+
+While video content is great, it isn’t always the best option for a public-facing asset, even when raw footage has already been recorded. If you aren't sure if a piece of content would be a good fit for video content, don't be afraid to ask!
+
+## Video Recording Best Practices
```

Comment:
```suggestion
## Setting Up your Recording Environment 
```
We don't generally use "best practices" as it's not aligned to the LP of "iteration" afaik. :)